zulint/printer.py

```diff
@@ -27,7 +27,7 @@ def print_err(name, color, line):
     if sys.version_info[0] == 3 and isinstance(line, bytes):
         line = line.decode('utf-8')
 
-    print('{}{}{}|{end} {}{}{end}'.format(
+    print('{}{}{}|{end} {}{!s}{end}'.format(
```

Comment:
For this, maybe we should rewrite this to use variable names for all the substitutions?  It's kinda hard to read as-is (took me a while to figure out which variable this goes with).

Reply:
Cool Updated!